bindings/swift/xcode/IotaWallet/IotaWallet.xcodeproj/project.pbxproj

```diff
@@ -14,7 +14,8 @@
 		7904C23327BBDFB2001CB01B /* Wallet.m in Sources */ = {isa = PBXBuildFile; fileRef = 7904C23027BBDFB2001CB01B /* Wallet.m */; };
 		7904C23827BBDFD2001CB01B /* await.swift in Sources */ = {isa = PBXBuildFile; fileRef = 7904C23627BBDFD2001CB01B /* await.swift */; };
 		7904C23927BBDFD2001CB01B /* callback.m in Sources */ = {isa = PBXBuildFile; fileRef = 7904C23727BBDFD2001CB01B /* callback.m */; };
-		7904C23C27BBE08D001CB01B /* libiota_wallet.dylib in Frameworks */ = {isa = PBXBuildFile; fileRef = 7904C23B27BBE08D001CB01B /* libiota_wallet.dylib */; };
+		79BD1BBB27CE54810043965C /* libiota_wallet.dylib in Frameworks */ = {isa = PBXBuildFile; fileRef = 79BD1BBA27CE54810043965C /* libiota_wallet.dylib */; };
```

Comment:
Is this all auto generated and is this file needed? Or could we just gitignore it so we don't always get these changes?

Reply:
Feel free to merge if it's good to keep the file

Reply:
This one is auto-generated based on Xcode settings I think, I'll add gitignore later on.